Gemfile

```diff
@@ -13,7 +13,7 @@ group :jekyll_plugins do
   gem 'jekyll-asset-pipeline', git: 'https://github.com/crdschurch/jekyll-asset-pipeline', tag: '0.0.2'
   # gem 'jekyll-asset-pipeline', path: File.expand_path('../jekyll-asset-pipeline', __dir__)
   gem "jekyll-contentful", git: 'https://github.com/crdschurch/jekyll-contentful.git', tag: '1.2.1'
-  gem "jekyll-crds", git: 'https://github.com/crdschurch/jekyll-crds.git', tag: '0.0.4'
+  gem "jekyll-crds", git: 'https://github.com/crdschurch/jekyll-crds.git', branch: 'feature/US17295-jekyll-updates'
```

Comment:
I will merge jekyll-crds before this goes in. Can you reference the proper version instead of a local branch here?